rdr_service/resource/schemas/participant.py

```diff
@@ -243,6 +243,27 @@ class Meta:
         pii_filter = {}  # dict(field: lambda function).
 
 
+class ParticipantParingHistorySchema(Schema):
```

Comment:
if there's a need to validate resource vs. BQ schema, the unittest infrastructure is in 
tests/resource_tests/schema_tests/test_resource_schemas.py

Reply:
I added this schema to the unittests.